### ISSUE TITLE
Layout main-X not subtracting border from rows/columns

### ISSUE THREAD

**Issue by Nycroth** [tmux/tmux#1873]
### Issue description
There appears to be an off by one error in the row/column count caused by the top/left border not being subtracted when the layout selected is `main-horizontal`/`main-vertical`.

To reproduce:
1. Open a new tmux session
2. `split-window` (any direction)
3. `select-layout main-vertical`
3. In right pane, run `python -c 'print("o"*1000)'`
4. Ensure one or more lines reaches the last column and wraps the terminal, not just pane

Same thing can be accomplished by reaching the last line of `main-horizontal` which will either scroll the terminal or overwrite the status line depending on its location.

![Screen Shot 2019-08-15 at 16 05 22](https://user-images.githubusercontent.com/6979898/63123230-816d9780-bf76-11e9-9783-de7d5db520e7.png)
![Screen Shot 2019-08-15 at 16 06 53](https://user-images.githubusercontent.com/6979898/63123316-b843ad80-bf76-11e9-8632-f51af7b6a520.png)

Confirmed in MacOS both iTerm2 and Terminal using no `~/.tmux.conf`.

### Required information
```
> uname -sp && tmux -V && echo $TERM
Darwin i386
tmux 2.9a
screen
```

* Logs from tmux (`tmux kill-server; tmux -dvv new`).
[tmux-client-5177.log](https://github.com/tmux/tmux/files/3506827/tmux-client-5177.log)
[tmux-out-5179.log](https://github.com/tmux/tmux/files/3506828/tmux-out-5179.log)
[tmux-server-5179.log](https://github.com/tmux/tmux/files/3506829/tmux-server-5179.log)

**Comment by Nycroth** [comment]
Just tested 3.0rc and this has been fixed.

**Comment by Nycroth** [comment]
Sorry for the open and close. Confirmed fixed in 3.0-rc4.
